js/element.js

```diff
@@ -14,3 +14,10 @@ export const $clearButton = document.getElementById("clear-button");
 export const $splashScreen = document.getElementById("splash-screen");
 export const $header = document.querySelector("header");
 export const $main = document.querySelector("main");
+export const $allFilter = document.getElementById("all-filter-button");
+export const $incompleteFilter = document.getElementById(
+    "incomplete-filter-button"
+);
+export const $completedFilter = document.getElementById(
+    "complete-filter-button"
+);
```

Comment:
```suggestion
export const $allFilterButton = document.getElementById("all-filter-button");
export const $incompleteFilterButton = document.getElementById(
    "incomplete-filter-button"
);
export const $completedFilterButton = document.getElementById(
    "complete-filter-button"
);
```

aren't they now more meaningful? We should declare the element name and variable name and function name more meaning so that other person can easily understand what type of variable, element and function it is.